src/styles/todo.scss

```diff
@@ -65,21 +65,21 @@
       color: #af5b5e;
     }
   }
-  
+
   &:hover &__remove {
```

Comment:
Consider using a nested class for the __remove element, like '.todo-item__remove'